content/SCALE/SCALEUIReference/Apps/InstallCustomAppScreens.md

```diff
@@ -0,0 +1,274 @@
+
+### Application Name Settings
+**Application Name** has two required settings, **Application Name** and **version**. 
+After completing the installation these settings are not editable.
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppApplicationName.png" alt="Application Name" id="3: Application Name" >}}
+
+{{< expand "Settings Information" "v" >}}
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Application Name** | Enter a name for the application. The name must have lowercase alphanumeric characters, begin with an alphabet character, and can end with an alphanumeric character. The name can contain a hyphen (-) but not as the first or last character in the name. For example, using *chia-1* but not *-chia1* or *1chia-* as a valid name. |
+| **Version** | Displays the current version of the default application. Enter the version of the application you want to install.|
+{{< /truetable >}}
+{{< /expand >}}
+
+### Container Images Settings
+**Container Images** settings specify the container image details. 
+They define the image tag, when TrueNAS pulls the image from the remote repository, how the container updates, and when a container automatically restarts.
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppContainerImages.png" alt="Container Images Settings" id="4: Container Images Settings" >}}
+
+{{< expand "Settings Information" "v" >}}
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Image Repository** | Required. Enter the Docker image repository name. For example, *plexinc/pms-docker* for Plex.|
+| **Image Tag** | Enter the tag to use for the specified image. For example, *1.20.2.3402-0fec14d92* for Plex. |
+| **Image Pull Policy** | Select the Docker image pull policy from the dropdown list. Options are **Only pull image if not present on host** (default option), **Always pull image even if present on host**, or **Never pull image even if it's not present on host**. |
+{{< /truetable >}}
+{{< /expand >}}
+
+### Container Entrypoint Settings
+**Container Entrypoint** settings specify both [commands and arguments](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/) to use for the image.
+These can override any existing commands stored in the image. 
+Check the documentation for the application you want to install for entry point commands or arguments you need to enter. 
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppContainerEntrypoint.png" alt="Container Entrypoint Settings" id="5: Container Entrypoint Settings" >}}
+
+{{< expand "Settings Information" "v" >}}
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Container CMD**| Click **Add** to display a **Command** field. |
+| **Command** | Enter container command. For example, if adding MinIO, enter *SERVER*. |
```

Comment:
Enter *a* container command, *the* container command, or container command*s* ?

Reply:
fixed